tests/cnt-i18n_xhtml-dir/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,22 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en" dir="rtl">
```

Comment:
Can you name this cnt-xhtml-dir, please?

I see that I named the anchor #confreq-rs-html-dir, and it was a weird choice to omit the "x"; feel free to update the spec to make it use "xhtml" too. It's a new anchor so no one should be using it yet.